### PR TITLE
coat_mini update

### PR DESCRIPTION
**Pull request by SunJuil-Ty**
Thank you for your contribution to the MindCV repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: configs/coat/README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Our reproduced model performance on ImageNet-1K is reported as follows.
 | coat_lite_tiny | D910x8-G | 77.35 | 93.43 | 5.72      | [yaml](https://github.com/mindspore-lab/mindcv/blob/main/configs/coat/coat_lite_tiny_ascend.yaml) | [weights](https://download.mindspore.cn/toolkits/mindcv/coat/coat_lite_tiny-fa7bf894.ckpt) |
 | coat_lite_mini | D910x8-G | 78.51 | 93.84 | 11.01 | [yaml](https://github.com/mindspore-lab/mindcv/blob/main/configs/coat/coat_lite_mini_ascend.yaml) | [weights](https://download.mindspore.cn/toolkits/mindcv/coat/coat_lite_mini-55a52f05.ckpt) |
 | coat_tiny | D910x8-G | 79.67 | 94.88 | 5.50 | [yaml](https://github.com/mindspore-lab/mindcv/blob/main/configs/coat/coat_tiny_ascend.yaml) | [weights](https://download.mindspore.cn/toolkits/mindcv/coat/coat_tiny-071cb792.ckpt) |
+| coat_mini | D910x8-G | 81.08 | 95.34 | 10.34 | [yaml](https://github.com/mindspore-lab/mindcv/blob/main/configs/coat/coat_mini_ascend.yaml) | [weights](https://download.mindspore.cn/toolkits/mindcv/coat/coat_mini_57c5bce7.ckpt) |
```
Comment (geniuspatrick) [inline configs/coat/README.md:20]: coat_mini-57c5bce7

Comment (geniuspatrick) [reply configs/coat/README.md:20]: 短横线

File: configs/coat/coat_mini_ascend.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ckpt_save_dir: './ckpt/'
+dataset_sink_mode: True
+amp_level: 'O2'
+
+# loss
+loss: 'CE'
+label_smoothing: 0.1
+
+# lr scheduler
+scheduler: 'warmup_cosine_decay'
```
Comment (geniuspatrick) [inline configs/coat/coat_mini_ascend.yaml:48]: cosine_decay即可

File: mindcv/models/coat.py [hunk 3]
```diff
@@ -45,7 +45,7 @@ def _cfg(url='', **kwargs):
 
 default_cfgs = {
     'coat_tiny': _cfg(url='https://download.mindspore.cn/toolkits/mindcv/coat/coat_tiny-071cb792.ckpt'),
-    'coat_mini': _cfg(url=''),
+    'coat_mini': _cfg(url='https://download.mindspore.cn/toolkits/mindcv/coat/coat_mini_57c5bce7.ckpt'),
```
Comment (geniuspatrick) [inline mindcv/models/coat.py:48]: 同上

**Review by geniuspatrick — CHANGES_REQUESTED** [review]

File: mindcv/models/coat.py (hunk 3 above)
Comment (SunJuil-Ty) [reply mindcv/models/coat.py]: 已修改

File: configs/coat/coat_mini_ascend.yaml (hunk 2 above)
Comment (SunJuil-Ty) [reply configs/coat/coat_mini_ascend.yaml]: 已修改

File: configs/coat/README.md (hunk 1 above)
Comment (SunJuil-Ty) [reply configs/coat/README.md]: 已修改

**Review by geniuspatrick — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**